config/initializers/rack/attack.rb

```diff
@@ -43,11 +43,8 @@ class Attack
     end
 
     def self.track_and_return_ip(req)
-      ip_address = if ApplicationConfig["FASTLY_API_KEY"].present?
-                     req.env["HTTP_FASTLY_CLIENT_IP"]
-                   else
-                     req.remote_ip
-                   end
+      ip_address = req.env["HTTP_FASTLY_CLIENT_IP"] || req.remote_ip
```

Comment:
If Fastly is utilized, you _will_ have `HTTP_FASTLY_CLIENT_IP`. I checked through Honeycomb to come to that conclusion